### PR TITLE
basic setup

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### Added
- basic setup lint and babel.
- init_setup.sh to install symbol-data-lib local project

### update
- symbol-sdk

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,16 +21,24 @@
...
+    "babel-eslint": "^10.1.0",
+    "babel-preset-env": "^1.7.0",
     "body-parser": "^1.18.3",
+    "cross-env": "^7.0.2",
     "express": "^4.14.0",
     "express-ws": "^4.0.0",
     "mongodb": "^2.2.16",
     "mongoose": "^5.5.8",
-    "nem2-sdk": "^0.16.3",
+    "symbol-data-lib": "file:symbol-data-lib",
```
Comment (OlegMakarenko) [inline package.json:36]: It can be installed directly from github
`"symbol-data-lib": "nemfoundation/symbol-data-lib"`

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "rules": {
+      "comma-dangle": ["error", "never"],
+      "no-control-regex": "off",
+      "no-empty": ["error", {
+          "allowEmptyCatch": true
+      }],
+      "curly": ["error", "multi-or-nest"],
+      "semi": ["error", "never"],
+      "indent": ["error", 2],
```
Comment (OlegMakarenko) [inline .eslintrc:20]: Do you mind removing `semi` rule and changing `indent` to 4?

Comment (AnthonyLaw) [reply .eslintrc:20]: sure thing.

File: package.json (hunk 1 above)
Comment (AnthonyLaw) [reply package.json:36]: `npm i nemfoundation/symbol-data-lib`
i tried, it's not working in my machine, that is error message.

i did a shell script `init_setup.sh` for this, you can have a look on it.

**Review by OlegMakarenko — APPROVED** [review]

File: .eslintrc (hunk 2 above)
Comment (OlegMakarenko) [reply .eslintrc]: thanks

File: package.json (hunk 1 above)
Comment (OlegMakarenko) [reply package.json:36]: Will take a look

**Pull request merged**